### PR TITLE
Add pt-BR support for the Examples page

### PR DESCRIPTION
**Pull request by trizotti**
Continuing Portuguese Brazil translation of the docs. (https://github.com/nuxt/nuxtjs.org/issues/1987)

### CODE REVIEW

**Review by smarroufin — APPROVED** [review]
Hi @trizotti 
That is a lot of work, thanks for the contribution! 💚 

LGTM! Just waiting for @ChristopheCVB approval and we are good to go.

Just so you know, and you probably noticed, PT/BR are not yet available on prod. Donc worry this will be live at some point, there are stuff that need to be taken care of at NuxtLabs before we put it live.

**Review by ChristopheCVB — APPROVED** [review]
Amazing work! Thanks!

**Pull request merged**
